05.ls/ls.rb

```diff
@@ -100,18 +97,39 @@ def display(files, row)
     end
 
     print cmod.ljust(12)
-    print fs.nlink.to_s.rjust(nlink_brank)
-    print ' '
-    print Etc.getpwuid(fs.uid).name.ljust(uid_brank)
-    print Etc.getgrgid(fs.gid).name.ljust(gid_brank)
-    print fs.size.to_s.rjust(filesize_brank)
-    print MONTH_TABLE[fs.mtime.to_a.slice(4).to_s].rjust(4)
-    print fs.mtime.to_a.slice(3).to_s.rjust(3)
-    print Time.now - fs.mtime < 15_552_000 ? fs.mtime.to_s.slice(11, 5).to_s.rjust(6) : fs.mtime.to_a.slice(5).to_s.rjust(6)
-    print ' '
+    l_disp_t(fs, nlink_blank, uid_blank, gid_blank, filesize_blank)
     print sorted_file
     puts
```

Comment:
これもいまさらですが、最後に改行するだけならこれでいいですね。
```suggestion
    puts sorted_file
```